R/specificity-plots.R

```diff
@@ -15,9 +15,7 @@
 #' 
 #' @export
 #' @examples
-#' \dontrun{
 #' signalMatrix = calcOpenSignal(vistaEnhancers, exampleOpenSignalMatrix_hg19)
```

Comment:
likewise, this will not run successfully because these objects are not there, are they?

Reply:
I think @stolarczyk made it so that we can call on them automatically. I've been running the R CMD check consistently and haven't had issues with that.

Reply:
yes, exampleOpenSignalMatrix_hg19 and vistaEnhancers are available in the environment upon GD package loading